packages/lib-panoptes-js/src/resources/collections/rest.js

```diff
@@ -0,0 +1,52 @@
+const panoptes = require('../../panoptes')
+const { endpoint } = require('./helpers')
+const { raiseError } = require('../../utilityFunctions')
+
+function create (params) {
+  const newCollectionData = (params && params.data) ? params.data : {}
+  const authorization = (params && params.authorization) ? params.authorization : ''
+  const projects = params && params.projects
+  const subjects = (params && params.subjects) ? params.subjects : []
+  if (!projects || !Array.isArray(projects)) return raiseError('Collection create: projects array is required.', 'error')
```

Comment:
Having a project link actually isn't required, although collections are mostly used in this way. A user can create a collection without a project link.